bluepysnap/circuit_ids.py

```diff
@@ -0,0 +1,222 @@
+# Copyright (c) 2020, EPFL/Blue Brain Project
+
+# This file is part of BlueBrain SNAP library <https://github.com/BlueBrain/snap>
+
+# This library is free software; you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License version 3.0 as published
+# by the Free Software Foundation.
+
+# This library is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+# details.
+
+# You should have received a copy of the GNU Lesser General Public License
+# along with this library; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+"""Circuit ids."""
+import numpy as np
+import pandas as pd
+import six
+
+from bluepysnap import utils
+from bluepysnap.exceptions import BluepySnapError
+
+
+class CircuitNodeIds:
+    """Global Node ids.
```

Comment:
CircuitNodeIds os fine for me. NodeIndex is too broad.

Reply:
I don t want "global' because this is too close to the gids from old format in bluepy
Index only is too broad because you can have "edge index" also.
I think I will stick to CircuitNodeIds/CircuitNodeId